docs/recipes/hitl.md

```diff
@@ -2,45 +2,49 @@
 layout: guide
 ---
 
-Botpress allows you to build a quite powerful tool for communication with user.
-But even with it there may be cases that are either impossible of very resource-consuming to implement and you may consider having an operator communicating certain users under some  conditions.
+Botpress allows you to build a powerful tool for autonomous communication with your users.
+However there may be cases where it is difficult or very resource-consuming to implement a conversation flow within the bot. At this point you may consider having a human take over the conversation and continue to communicate with your user.
 
-[@botpress/hitl](https://github.com/botpress/botpress/tree/master/packages/functionals/botpress-hitl) is designed to solve this issue (only supports messenger-channel for now).
+The [Human-in-the-Loop (@botpress/hitl)](https://github.com/botpress/botpress/tree/master/packages/functionals/botpress-hitl) module allows you to do just that! (It does only support messenger-channel for now).
```

Comment:
It has been updated to support `channel-web` too I think? @epaminond

Reply:
I think yes, I believe it worked on channel-web for me when I tested it a few months ago.

Reply:
I have updated this for your review